idaes/power_generation/carbon_capture/mea_solvent_system/property_package/liquid_prop.py

```diff
@@ -0,0 +1,1282 @@
+                        StateBlockData,
+                        StateBlock,
+                        Component,
+                        LiquidPhase)
+
+from idaes.core.util.initialization import (fix_state_vars,
+                                            revert_state_vars,
+                                            solve_indexed_blocks)
+from idaes.core.util.misc import add_object_reference
+from idaes.core.util.model_statistics  import (degrees_of_freedom,
+                                              number_unfixed_variables,
+                                              report_statistics)
+import idaes.logger as idaeslog
+
+__author__ = "Paul Akula, John Eslick"
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+@declare_process_block_class("LiquidParameterBlock")
+class PhysicalParameterData(PhysicalParameterBlock):
+    """
+    Liquid Phase Property Parameter Block Class
+
+    Contains parameters and indexing sets associated with
+    liquid phase properties for amine-based scrubbing process.
+
+    """
+    CONFIG = PhysicalParameterBlock.CONFIG()
+    CONFIG.declare("process_type", ConfigValue(
+        default='Absorber',
+        domain=In(['Absorber', 'Stripper','Regenerator']),
+        description="Flag indicating the type of  process",
+        doc="""Flag indicating either absoprtion or stripping process.
+               Heat of absorption depends on teh process type """))
+
+    def build(self):
+        '''
+        Callable method for Block construction.
+        '''
+        super(PhysicalParameterData, self).build()
+
+        self._state_block_class = LiquidStateBlock
+
+        # Create Phase object
+        self.Liq = LiquidPhase()
+
+        # Create Component objects
+        self.MEA = Component()
+        self.CO2 = Component()
+        self.H2O = Component()
+
+        #create component list for true species
+        self.component_list_true = Set(initialize=
+                             ['CO2', 'H2O', 'MEA','MEA+','MEACOO-', 'HCO3-'])
+        self.component_list_anion = Set(initialize=['MEACOO-', 'HCO3-'])
+        self.component_list_cation = Set(initialize=['MEA+'])
+        self.component_list_solvent = Set(initialize=['H2O', 'MEA'])
+        self.component_list_diffus = Set(initialize=
+                                          ['CO2','MEA','MEA+','MEACOO-'])
+        #Reaction index
+        self.reaction_idx = Set(initialize=['R1','R2'])
+
+        # Thermodynamic reference state
+        self.pressure_ref = Param(within=PositiveReals,
+                                  mutable=True,
+                                  default=101325,
+                                  doc='Reference pressure [Pa]')
+        self.temperature_ref = Param(default=298.15,
+                    doc='Thermodynamic Reference Temperature [K]')
+
+        # Critical Temperature dictionary, unit K (Aprion 2005)
+        T_crit_dict = {'CO2':304.18,'H2O':647.13,'MEA':614.45}
```

Comment:
Yes, for the liquid phase it can be directly initialized. However,  like the vapor phase where there is a need for a "main" dictionary that holds the parameters for all known system components, only a subset of the "main" dictionary is then initialized depending on the components in the component list. This is clearer in the vapor phase package.

Reply:
That does not apply in this case however - whilst it might make sense in another package, there is no need for it here.

Reply:
```dict``` is now  put directly to initialize the  ```Param```